### ISSUE TITLE
Searched districts sometimes do not appear when switching states

### ISSUE THREAD

**Issue by amygroshek** [Hyperobjekt/react-seda-scatterplot#11]
I'm making a note of this here and plan to investigate further, but certain districts, when displayed as the search result, don't show up in certain states (and do show up in others). My home district, for example, "Adams-Friendship School District, Wisconsin" disappears when scrolling from state 3 to state 4 in this article:
https://eop-staging.netlify.com/posts/the-opportunity-gap-2019-01-12/

Other districts of similar size/location do appear. 

While related to the search component, this may be/probably is ultimately caused by/related to the scatterplot, so I'm logging it here.

**Comment by Lane** [comment]
Some of the places don't have data for some of the metrics. So, if data isn't available for a location, it is dropped from the scatterplot.

I just checked out "Adams-Friendship School District", which has an ID of 5500060.  In the [socioeconomic status white / black gap data](https://d2fypeb6f974r1.cloudfront.net/dev/scatterplot/districts/wb_ses.csv), there is no data value for 5500060.

Should we add some sort of "data unavailable" notification?  Not sure if that's something we should integrate into the scatterplot or handle outside of it.

**Comment by amygroshek** [comment]
Makes sense, @Lane , James and I were just looking at it earlier and assumed it was data-related. @james-minton , would this be a good question for the client?

**Comment by james-minton** [comment]
@amygroshek, agreeing with some kind of "no data" notice for these places. Where that lives in the tool is another question. Needs some thought / discussion. Meanwhile will let client know about the issue and get their input too. Glad you discovered it!

**Comment by amygroshek** [comment]
Agree with @Lane, how to handle it is tricky. It's the scatterplot component that will "know" that there's no data. But if we only handle it using the scatterplot, showing a notification, then the search component, will still display that district name as though it's selected. And ideally the search input resets and displays its search prompt again. 

This issue also means that it's currently possible to search for items in the search component that won't appear on the plot/aren't in that plot state data set... Ideally the search prompts don't contain districts that are not in the current state's data... If we do that we need to integrate the two components and the state switching more tightly, or at least build in some additional callbacks. And there might be cases where we initially present the user with fewer search results than will be shown in subsequent plots with larger data sets...
